src/main/java/FC/Controller.java

```diff
@@ -92,9 +120,15 @@ private void removeFromMenu(String table) throws SQLException {
     /**
      * Add card to an existing table/deck
      */
-    private void insertCard(String table, String term, String def) throws SQLException{
-            String insertCardStmt = "INSERT INTO " + table + "VALUES (" + term + ", " + def + ");";
-            executeCUD(insertCardStmt);
+    //combined both and made public
+    public void insertCard(String table, String term, String def) throws SQLException {
+        editDeck = getDeck(table);
+        String insertCardStmt = "INSERT INTO " + table + "VALUES (" + term + ", " + def + ");";
+        executeCUD(insertCardStmt);
+        //fix what id is
+        String id = "";
```

Comment:
The database should generate this

Reply:
DB should be queried to get the card bc we can't assign the id manually

Reply:
We do need a seperate method bc when they're creating a new deck from scratch, there's no "editDeck", unless we essentially make them the same processes